src/core/sec.rs

```diff
@@ -74,9 +74,9 @@ enum AfInet {
 fn rdp_extended_infos(performance_flags: u32) -> Component {
     component![
         "clientAddressFamily" => U16::LE(AfInet::AfInet as u16),
-        "cbClientAddress" => DynOption::new(U16::LE(0), |x| MessageOption::Size("clientAddress".to_string(), x.inner() as usize + 2)),
+        "cbClientAddress" => U16::LE(2),
```

Comment:
What exactly was wrong here?

Reply:
I'm not exactly sure what was meant to happen there, but it seems like it was something like: calculate the size of the string in the `"clientAddress"` field below and add 2 for the two null-terminator bytes.

Reply:
I feel like I may have written this to fix a security issue. I'll check the git blame tomorrow.

Reply:
I just checked -- no, it was citronneur. You only wrote the line
```rust
"performanceFlags" => U32::LE(performance_flags)
```

I don't see any potential security bugs here, we're just passing hardcoded flags to the RDP server.